client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content/index.tsx

```diff
@@ -270,7 +277,8 @@ export default function SiteStatusContent( {
 
 		if ( type === 'boost' ) {
 			const overallScore = rows.site.value.jetpack_boost_scores.overall;
-			if ( hasBoost ) {
+
+			if ( overallScore ) {
```

Comment:
We now move this outside the `if ( hasBoost )` scope as we now display 1-time scores with sites that do not have **Jetpack Boost** plugin installed.